src/curated-corpus/components/actions/ScheduleCorpusItemAction/ScheduleCorpusItemAction.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { useNotifications, useRunMutation } from '../../../../_shared/hooks';
+import {
+  ApprovedCorpusItem,
+  CreateScheduledCorpusItemInput,
+  useCreateScheduledCorpusItemMutation,
+} from '../../../../api/generatedTypes';
+import { FormikHelpers, FormikValues } from 'formik';
+import { ScheduleItemModal } from '../../ScheduleItemModal/ScheduleItemModal';
+import { DateTime } from 'luxon';
+
+interface ScheduleCorpusItemActionProps {
+  /**
+   * The approved item that is to be scheduled onto a surface.
+   */
+  item: ApprovedCorpusItem;
+
+  /**
+   * A state variable that tracks whether the ScheduleItemModal is visible
+   * on the page or not.
+   * This has to be passed down from the parent component as other components
+   * may need to use it, too.
+   */
+  modalOpen: boolean;
+
+  /**
+   * A function that toggles the ScheduleItemModal's visibility on and off.
+   * This has to be passed down from the parent component as other components
+   * may need to use it.
+   */
+  toggleModal: VoidFunction;
+
+  /**
+   * A helper function from Apollo Client that triggers a new API call to refetch
+   * the data for a given query.
+   */
+  refetch?: VoidFunction;
+}
+
+/**
+ * This component encapsulates the logic needed to reject a corpus item -
```

Comment:
Minor: just a typo here we still have `reject` instead of `schedule`

Reply:
Fixed, now need a re-approval :)